tycho-spi/src/main/resources/org/eclipse/tycho/version/version.properties

```diff
@@ -3,5 +3,4 @@ bnd=${bnd.version}
 scm=${env.GIT_COMMIT}
 url=${env.GIT_URL}
 branch=${env.GIT_BRANCH}
-scmBranch=${scmBranch}
```

Comment:
It looks like this is no longer in use so it seems safe to remove but I wonder why it affects reproducibility?

Reply:
when you `git checkout` a git tag, what is the current branch?